module/programs/helper.js

```diff
@@ -1045,10 +1046,11 @@ module.exports = class ProgramsHelper {
 	 * @name userPrivatePrograms
 	 * @param {String} userId
 	 * @param {String} language -languageCode
+	 * @param {Boolean} projectsCount - get the projectsCount under that program.
 	 * @returns {JSON} - List of programs that user created on app.
 	 */
 
-	static userPrivatePrograms(userId, language = '') {
+	static userPrivatePrograms(userId, language = '', projectsCount = false) {
 		return new Promise(async (resolve, reject) => {
 			try {
 				let programsData = await programsQueries.programsDocument(
```

Comment:
@praveenKDass check if we can apply isDeleted flag as well

Reply:
added query based on isDeleted as well @VISHNUDAS-tunerlabs